designsafe/static/scripts/projects/components/edit-project/edit-project.component.js

```diff
@@ -424,25 +421,48 @@ class EditProjectCtrl {
         if (typeof this.form.keywords !== 'undefined') {
             projectData.keywords = this.form.keywords;
         }
+        if (!projectData.teamMembers.concat(projectData.coPis, [projectData.pi]).includes(this.form.creator.username) && this.form.uuid) {
+            this.modalInstance = this.$uibModal.open({
+                component: 'confirmMessage',
+                resolve: {
+                    message: () => "Are you sure you want to remove yourself from the project?",
+                },
+                size: 'sm'
+            });
+            this.modalInstance.result.then((res) => {
+                if (!res) {
+                    this.ui.busy = false;
+                } else {
+                    this.savePrj(projectData).then((project) => {
+                        if (this.project) {
```

Comment:
This function might also need to forward the user back to the projects listing.  As it stands now, after I remove myself from a project I'm returned to the project.  And, I still have the ability to add myself back as a PI.  It's not until I go to the projects listing that I'm unable to access the project anymore.

Reply:
Ah good point

Reply:
It works!